sovrin_client/agent/walleted.py

```diff
@@ -97,6 +103,17 @@ def __init__(self,
         }
         self.logger = logger
 
+        self.issuer_backend = None
+
+        self._invites = {}  # type: Dict[Nonce, Tuple(InternalId, str)]
+        self._attribDefs = {}  # type: Dict[str, AttribDef]
+        self.defined_claims = []  # type: List[Dict[str, Any]
+
+        self.available_claims_by_internal_id = {}  # type: Dict[InternalId, Set[ID]]
```

Comment:
I agree that persistence is something we want but these things are not wallet items. So I think they are fine as an in-memory object for now. We need an architecture for how we want to deal with this kind of data.  Right now, the creation of Acme, Faber and Thrift interact with them directly on creation. But I think we need a better story for them but I don't think the wallet is the right answer.